toolkit/offchain/src/csl.rs

```diff
@@ -610,6 +691,42 @@ mod tests {
 		assert_eq!(ex_units.mem(), 1000u64.into());
 		assert_eq!(ex_units.steps(), 2000u64.into());
 	}
+
+	#[test]
+	fn get_validator_budgets_works() {
+		let result = get_validator_budgets(vec![
+			OgmiosEvaluateTransactionResponse {
+				validator: OgmiosValidatorIndex::new(1, "mint"),
+				budget: OgmiosBudget::new(11, 21),
+			},
+			OgmiosEvaluateTransactionResponse {
+				validator: OgmiosValidatorIndex::new(0, "not mint"),
```

Comment:
Now, if someone alters implementation and will match for "spend", that will be equally good implementation as matching for "mint" this test will fail, despite implementation is as good as this one.
Is there a reason for not using real data if it is costs as much as unrealistic data?

Reply:
alright, changed that to "spend"

Reply:
Have you?

Reply:
yeah, forgot to push. fixed x)